labgrid/pytestplugin/reporter.py

```diff
@@ -21,6 +21,40 @@ def bold(text):
 def under(text):
     return "\033[4m{}\033[0m".format(text)
 
+@pytest.hookimpl()
+def pytest_collection_modifyitems(config, items):
+    """This function matches function feature flags with those found in the
+    environment and disables the item if no match is found"""
+    have_feature = []
+    env = config._labgrid_env
+
+    if not env:
+        return
+
+    have_feature = set(env.get_features() + env.get_target_features())
+
+    for item in items:
+        want_feature = []
+        marker = item.get_marker("lg_feature")
+        if marker:
+            arg = marker.args[0]
+            if isinstance(arg, str):
+                want_feature.append(arg)
```

Comment:
As you're not looping here, you can just assign want_feature directly. No need to create it as a empty list first.

Reply:
I need to initialize the variable, otherwise I'll get an error while calculating the missing features.